utils/utils.jsx

```diff
@@ -1564,6 +1534,7 @@ export function getRootId(post) {
     return post.root_id === '' ? post.id : post.root_id;
 }
 
+// NOTE this would be a sprawling refactor
```

Comment:
I'm pretty sure there's another ticket around for this that I filed a while back, so definitely feel free to skip this one.

Reply:
Ticket for reference: https://mattermost.atlassian.net/browse/MM-13303